example/index.js

```diff
@@ -5,7 +5,10 @@ const syncComplete = (res) => {
 bopen.addEventListener('click', event => {
 
   window.ethereum.enable().then(addresses => {
-    window.Box.openBox(addresses[0],  window.ethereum, {}).then(box => {
+    const idw = new IdentityWallet({ seed: '0x9827348745987235978347853475347345768349872384'})
```

Comment:
may make example confusing to have this as default

Reply:
Thanks, this should be reverted!